configuration/configuration_blackbox_test.go

```diff
@@ -0,0 +1,108 @@
+package configuration
```

Comment:
Sure, but the point of black box tests are to ensure that the correct methods are exposed and usable from other packages. 

Changing the package name to *_test simulates that :)

I've opened this PR which demonstrates how these methods would be used from external packages when needed.
https://github.com/almighty/almighty-core/pull/817